### ISSUE TITLE
Replace 4xx.html and 5xx.html with a single error.html page?

### ISSUE THREAD

**Issue by Rich-Harris** [sveltejs/sapper#209]
For me these files are normally identical. Maybe that's not true for everyone?

Also, it might need a 'special' name that isn't just `error.html`, since you might want to use that as a route.

**Comment by DanielSokil** [comment]
@Rich-Harris 
Is there a way to render an error page?
Ex.
```
export default {
  components: {
    Layout
  },

  preload({ params, query }) {
    // the `slug` parameter is available because this file
    // is called [slug].html
    const { slug } = params;

    let api = `//192.168.44.62:8000/${slug || "/"}/?api`;

    return fetch(api)
      .then(r => r.json())
      .then(post => {
        if (post.name === "notfound.md") {
          goto("error_page");
        } else {
          return {
            post
          };
        }
      });
  }
};
```

**Comment by DanielSokil** [comment]
```
return this.redirect(404, '/error');
```

**Comment by Rich-Harris** [comment]
This is fixed in 0.14
